code/modules/shuttle/shuttle.dm

```diff
@@ -777,8 +778,11 @@
 		switch(shuttle_master.moveShuttle(shuttleId, href_list["move"], 1))
 			if(0)
 				to_chat(usr, "<span class='notice'>Shuttle received message and will be sent shortly.</span>")
+				log_game("[key_name(usr)] called a shuttle on [src.name]")
 			if(1)
 				to_chat(usr, "<span class='warning'>Invalid shuttle requested.</span>")
+			if(2)
+				to_chat(usr, "<span class='warning'>Shuttle's on cooldown!</span>")
```

Comment:
Doesn't mater much at this point, but I figure a little fluff here to realize this would be nice.

`Shuttle's navigation systems are still re-calibrating from the last [alert.is_red() ? "emergency" : ""] dispatch.`

Super pseudocode check for alert level but it's a start.

Reply:
I agree with this, but please, don't add the alert level to this.